### DISCUSSION TITLE
Intentional community references - the long view,1970s onwards

### DISCUSSION THREAD

**Discussion by baouroux** [life-itself/community#158] [General]
Wonderful to connect to the community here, and as a first post, sharing here below the references on intentional communities that have stood out for me:

The rich biography of William Irwin Thompson, and his experience of the journey with the **Lindisfarne** community, great insights, including his association with a buddhist community: "Thinking Together At The Edge Of History: A Memoir of the Lindisfarne Association" 2016 

The thought leadership is still so relevant today I feel....including with the audios below:

- A super audio resource of the Lindisfarne events, hosted by the US Schumacher Center: https://centerforneweconomics.org/envision/legacy/lindisfarne-tapes/
- Including iconic presentations (for me) from Schumacher himself....I read Small is Beautiful while at university, and it changed my career path from working for my aerospace sponsor company through my studies, to working for 4 years for the development aid charity that Schumacher founded!
- It is fascinating to see how the Lindisfarne community tackled such contemporary topics in the 1970s, and that the community was such a unique hub of intellectual inquiry and discourse.

Rosabeth Kanter presents at one of the Lindisfarne conferences, her research on intentional communities: "The author of Community and Commitment analyzes the factors that make a difference between success and failure in alternative communities.: https://archive.org/details/RosabethKanterG8

You might have heard of the intentional community Possibility Alliance in Maine US? I think is it quite a small community...I was really touched by the story of Ethan Hughes: https://www.awakin.org/v2/calls/617/ethan-hughes/recording?check=1

**Comment by rufuspollock** [comment]
@baouroux thanks for the great suggestions. you may also be interested in this related topic https://github.com/life-itself/community/discussions/144 about creating a database of intentional communities.

Re Lindisfarne: really interested in Lindisfarne for some time and see it as inspiration for Life Itself (see partial wiki entry https://github.com/life-itself/community/blob/main/notes/lindisfarne-association.md). Love to learn more .

> Thinking Together At The Edge Of History: A Memoir of the Lindisfarne Association

I've wanted to get a copy of this. Do you have any notes, highlights or insights you could share?

> The thought leadership is still so relevant today I feel....including with the audios below:

We could add this material to the wiki page if you were interested to get further into the geekery here! https://github.com/life-itself/community/blob/main/notes/lindisfarne-association.md

> Rosabeth Kanter presents at one of the Lindisfarne conferences, her research on intentional communities: "The author of Community and Commitment analyzes the factors that make a difference between success and failure in alternative communities.: https://archive.org/details/RosabethKanterG8

I've added this now to my reading list on "alternative / intentional communities" (what's the standard term?). We could create a section on this in zotero.

> You might have heard of the intentional community Possibility Alliance in Maine US? I think is it quite a small community...I was really touched by the story of Ethan Hughes: [awakin.org/v2/calls/617/ethan-hughes/recording?check=1](https://www.awakin.org/v2/calls/617/ethan-hughes/recording?check=1)

I hadn't heard of them. One to add to the db perhaps?
